.github/workflows/docker.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       packages: write
 
     outputs:
-      tags: steps.meta.outputs.tags
+      tags: steps.tag.outputs
```

Comment:
do `run` steps just put their stdout into outputs as a string?

Reply:
no, the output is the `GITHUB_OUTPUT` env-var, but I messed up the format and am fixing that right now